index.html

```diff
@@ -0,0 +1,243 @@
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>           
+                
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>    
+                                
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>
+
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Datas</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>            
+            </div>
+        
+        
+    </section>
+    
+    <section id="skillset">
+        <div class="about">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me..</p>
+                    <div class="orange-button2">
+                        <div class="see-project2">Get my resume</div>
```

Comment:
- [ ] Please 🙏🏻  follow the best practices by using correct semantics in your *HTML* code. As I can see that you have used the `div` tag instead of the **button** tag which is not good at all because `div` is not a semantics tag only used for `decoration` purposes whereas on the other hand `button` is a semantics tag and have the meaning, for example, **Get my resume** should be a button so that later on when any person clicks on it then he/she will be able to download or see your *Resume* 🗒️.